src/main/steps/common/form/fields/checkboxes.njk

```diff
@@ -2,23 +2,23 @@
 
 {% for checkbox in field.values %}
   {# Required for sending an empty value when the checkbox has not been checked #}
-  <input type="hidden" value="" name="{{ checkbox.name }}">
+  <input type="hidden" value="unchecked" name="{{ checkbox.name }}">
```

Comment:
was there an issue with it being empty? just curious as in my multiple checkbox implementaiton, i get rid of empty values as this input value also is returned making it an array ['unchecked', 'checked'].

Reply:
👍 I've reverted this and kept `""` as unchecked.